apps/full-site-editing/full-site-editing-plugin/full-site-editing/blocks/navigation-menu/edit.js

```diff
@@ -0,0 +1,43 @@
+/* eslint-disable wpcalypso/jsx-classname-namespace */
+/**
+ * External dependencies
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { IconButton, ServerSideRender, Toolbar } from '@wordpress/components';
+import { BlockControls } from '@wordpress/editor';
+import { addQueryArgs } from '@wordpress/url';
+import { Fragment } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+
+const NavigationMenuEdit = ( { attributes } ) => {
+	const redirectToCustomizer = () => {
+		const url = addQueryArgs( 'customize.php', {
+			'autofocus[panel]': 'nav_menus',
+			return: window.location.href,
+		} );
+		window.location.href = url;
+	};
+	return (
+		<Fragment>
+			<BlockControls>
+				<Toolbar>
+					<IconButton icon="edit" label={ __( 'Edit Menu' ) } onClick={ redirectToCustomizer } />
```

Comment:
In order to improve the accessibility, I'd suggest to use a `href` property instead of `onClick`. See https://github.com/Automattic/wp-calypso/pull/33714#discussion_r291789685.

Reply:
How does it improve accessibility – more semantic you mean?.. I've committed an update here, but I admit there's much more robust styling for free by keeping it a button. I can go either way ¯\_(ツ)_/¯

Reply:
It actually improve both, but I consider this more beneficial for accessibility. Screen readers identify links to let users know that these elements will navigate them to a different page. Buttons have a general purpose and the screen readers used by people with vision disability won't know what they will do.

The necessity to add more styles for a link is not ideal, but I still consider is important to maintain a good accessibility if we can.

Reply:
The core project does use an `onClick` in addition to aria-pressed attribute https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/button_role

Reply:
467b6bbe5b1c6a4553b678bf9a109c6ec4260e13